node-red-contrib-ms-language/node-luis.js

```diff
@@ -12,61 +119,21 @@ module.exports = function(RED) {
 
         let node = this;
         let conf = RED.nodes.getNode(config.config);
-        node.status({fill:"red", shape:"ring", text: 'Missing credentials'});
-        
+        node.status({fill:'red', shape:'ring', text: 'Missing credentials'});
+
         if (!conf || !conf.credentials) return;
-        if (conf.way === "key") {
-            this.endpoint  = "https://" + (conf.location || "westus") + ".api.cognitive.microsoft.com/luis/v2.0/apps/" + conf.credentials.appId;
-            this.endpoint += "?subscription-key=" + conf.credentials.subKey + (conf.verbose ? "&verbose=" + conf.verbose : "") 
-            this.endpoint += (conf.staging ? "&staging=" + conf.staging : "") + "&q=";
+        if (conf.way === 'key') {
+            // Endpoint URL changes for V3
+            this.endpoint = `https://${conf.location}.api.cognitive.microsoft.com/luis/prediction/v3.0/apps/${conf.credentials.appId}/slots/${conf.staging ? 'staging' : 'production'}/predict?verbose=${conf.verbose ? 'true' : 'false'}&log=true&show-all-intents=true&query=`;
+            this.subKey = conf.credentials.subKey;
```

Comment:
Here, the subKey is provided only in "key" mode. I don't think that the "endpoint" mode still work.